.github/workflows/build.yml

```diff
@@ -99,22 +99,37 @@ jobs:
 # MAVEN BUILD START
 ##########################
 
-      # Phase 1 - Build and deploy to judong nexus
-      - name: 🔥 Build with Maven (build, test)
+      - name: 🔥 Build Maven project with tests (HSQLDB)
         run: |-
           ./mvnw -B -Dstyle.color=always \
           -Drevision=${{ steps.version.outputs.version }} \
-          -Psign-artifacts \
-          -Prelease-judong \
+          -Ddialect=hsqldb \
+          clean install
+
+      - name: 🔥 Test with PostgreSQL
+        run: |-
+          ./mvnw -B -Dstyle.color=always \
+          -Drevision=${{ steps.version.outputs.version }} \
+          -Ddialect=postgresql \
+          -Dskip-test-source-generation \
+          -f judo-runtime-core-jsl-itest \
+          test
+
+      - name: 🎁 Deploy with Maven to judong
+        run: |-
+          ./mvnw -B -Dstyle.color=always \
+          -Drevision=${{ steps.version.outputs.version }} \
+          -Ddeploy-only \
+          -Dmaven.test.skip=true \
+          -P"sign-artifacts,release-judong" \
```

Comment:
Have to avoid this hack.

Reply:
[a56f6eb](https://github.com/BlackBeltTechnology/judo-runtime-core-jsl/pull/97/commits/a56f6eb4c92d995a97574b0f499d0149c997cf08) [a20db73](https://github.com/BlackBeltTechnology/judo-runtime-core-jsl/pull/97/commits/a20db73e0cbfb3228bd0b7c98a68aa3d75789b4d)